plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeDelete.java

```diff
@@ -270,6 +270,19 @@ public void testDeleteMultiplePartitionKeys()
         assertQuery("SELECT * FROM " + tableName, "VALUES (1, 2, 4), (3, 2, 1), (null, null, null), (1, 1, 1)");
     }
 
+    // This is not the best class for this test but it doesn't fit any other places
+    // and here entire test infrastructure is present and ready
+    @Test
+    public void testQueryingNullPartitions()
```

Comment:
`testQueryNullPartitionWithNonPushdownablePredicate` + add a comment that non-pushdownable means not covertiabe to a TupleDomain